packages/oasis-actions/src/helpers/acceptedFeeToken.ts

```diff
@@ -0,0 +1,28 @@
+import { acceptedTokenAddresses, acceptedTokenSymbols } from '../config/acceptedFeeTokensConfig'
+
+type TokenSymbolOrAddress = string
+
+interface Props {
+  fromToken: TokenSymbolOrAddress
+  toToken: TokenSymbolOrAddress
+}
+
+/**
+ * Prefers sourceToken over targetToken
+ * Accepts args as either a token symbol or in address format
+ */
+export function acceptedFeeToken({ fromToken, toToken }: Props) {
+  if (acceptSourceToken(fromToken)) return 'sourceToken'
+  if (acceptTargetToken(toToken)) return 'targetToken'
+
+  const fallbackTokenType = 'sourceToken'
+  return fallbackTokenType
+}
+
+function acceptSourceToken(sourceToken: TokenSymbolOrAddress) {
+  return acceptedTokenSymbols.includes(sourceToken) || acceptedTokenAddresses.includes(sourceToken)
```

Comment:
if `acceptedTokenSymbols` and `acceptedTokenAddresses` are in the correct order then we can return an index of the given token. 
then `acceptedFeeToken` needs to return the token with a smaller number.